src/scss/index.test.scss

```diff
@@ -97,27 +97,6 @@ $tokens: (
 		}
 	}
 
-	@include it("it returns only alias tokens") {
-		@include assert {
-			@include output {
-				@include root.root-tokens(
-					(
-						alias: (
-							color: #f00,
-							font-family: "Inter",
-						),
-					)
-				);
-			}
-			@include expect {
-				:root {
-					--color: #f00;
-					--font-family: "Inter";
-				}
-			}
-		}
-	}
-
```

Comment:
@malavikakoppula , we should be keeping this test, was there a reason it was removed? I added it back in and it does pass just fine.